src/ldf-client-ui.js

```diff
@@ -350,11 +351,12 @@ if (typeof global.process === 'undefined')
                   $idp.show();
                 }
                 else {
+                  const href = window.location.href.replace('#', '?');
                   $solidSession.login({
                     oidcIssuer: $idp.val(),
-                    redirectUrl: window.location.href.replace('#', '?'), // OIDC does not allow hash fragments, so we encode it as query param
+                    redirectUrl: href, // OIDC does not allow hash fragments, so we encode it as query param
                     clientName: 'Comunica Web Client',
-                    clientId: 'https://query.linkeddatafragments.org/solid-client-id.json',
+                    clientId: join(href, 'solid-client-id.json'),
```

Comment:
We may not need this new dependency, it may be achievable using `relative-to-absolute-iri`